requirements.txt

```diff
@@ -1,5 +1,5 @@
 django>=1.11,<2.2
 django-leaflet>=0.23,<0.25
 channels>=2.1.3,<2.4.0
-Pillow>=4.3.0,<6.2
+Pillow>=6.0.0,<=6.2.1
```

Comment:
change this line to `Pillow>=6.0.0,<=6.3.0`, because we want to support all the 6.2.x versions.